ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -296,6 +307,9 @@ public Builder parseArg(String arg) throws IOException {
           case "maxInterval":
             maxInterval = parseIterations(argValue);
             break;
+          case "schemaregistryurl":
```

Comment:
Should be camel case to be consistent with the other arguments.

Reply:
Updated the name.